dev/io.openliberty.wsoc.2.2.internal_fat/fat/src/io/openliberty/wsoc22/tests/Spec185GetSessionTest.java

```diff
@@ -0,0 +1,134 @@
+
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.wsoc22.tests;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.util.logging.Logger;
+
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.ws.fat.util.browser.WebBrowser;
+import com.ibm.ws.fat.util.browser.WebBrowserFactory;
+import com.ibm.ws.fat.util.browser.WebResponse;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.HttpUtils;
+
+import org.asynchttpclient.Dsl;
+import org.asynchttpclient.ws.WebSocketUpgradeHandler;
+import org.asynchttpclient.ws.WebSocketListener;
+import org.asynchttpclient.ws.WebSocket;
+
+/**
+ * Verifies SendResult#getSession works per Spec #185
+ * https://github.com/jakartaee/websocket/issues/185
+ */
+@RunWith(FATRunner.class)
+public class Spec185GetSessionTest {
+
+    @Server("sessionTestServer")
+    public static LibertyServer LS;
+
+    private static final Logger LOG = Logger.getLogger(Spec185GetSessionTest.class.getName());
+
+    private static final String SESSION_WAR_NAME = "session";
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+
+        // Build the war app and add the dependencies
+        ShrinkHelper.defaultDropinApp(LS, SESSION_WAR_NAME + ".war", "io.openliberty.wsoc.spec185");
+
+        LS.startServer();
```

Comment:
It's a good idea to start the server with a name, such as `LS.startServer(Spec185GetSessionTest.class.getSimpleName() + ".log");` Then if we have another future test using the same Server, we'll have an easy way to look at the logs and know which test ran on what set of server logs we're looking at. Ideally, we'll have one test class on one server, but it doesn't hurt to be a bit verbose here.